modules/installation-rhv-building-ignition-files.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_rhv/installing-rhv-user-infra.adoc
+
+[id="installation-rhv-building-ignition-files_{context}"]
+= Building the ignition files
+
+To build the Ignition files from the manifest files you just generated and modified, you run the installation program. This action creates a {op-system-first} machine, `initramfs`, which fetches the ignition files and performs the configurations needed to bring a node to life.
+
+In addition to the Ignition files, the installation program generates the following:
+
+* An `auth` directory that contains the admin credentials for connecting to the cluster with the `oc` and `kubectl` utilities.
+* A `metadata.json` file that contains information such as the OCP cluster name, OCP cluster ID, and the `infraID` for the current installation.
+
+The Ansible playbooks for this installation process use the value of `infraID` as a prefix for the virtual machines they create. This prevents naming conflicts when there are multiple installations in the same oVirt/{rh-virtualization} cluster.
+
+[IMPORTANT]
+====
+Certificates in Ignition configuration files expire after 24 hours. Complete the cluster installation and keep the cluster running in a non-degraded state for 24 hours so that the first certificate rotation can finish.
+====
+
+.Procedure
+
+. To build the ignition files, enter:
++
+[source,terminal]
+----
+$ openshift-install create Ignition-configs --dir $ASSETS_DIR
+----
++
+.Example output
+[source,terminal]
+----
+$ tree
+.
+└── wrk
+    ├── auth
+    │   ├── kubeadmin-password
+    │   └── kubeconfig
+    ├── bootstrap.ign
+    ├── master.ign
+    ├── metadata.json
+    └── worker.ign
+----
+
+.Additional resources
+
+* link:http://coreos.github.io/ignition/[The CoreOS Ignition documentation]
```

Comment:
I'm not sure this is needed, as we don't want customers directly modifying initiation configs. IE we want them using the ones we generate, or updating the configs using manifest changes. 

cc: @control-d